bolt-design-system/bolt
# Fix: Improve Handling of Github API Requests

## Jira
N/A

## Summary
Update to better handle situation when the aggregated data about the latest tagged versions of Bolt has expired however the user / machine doesn't have a `GITHUB_TOKEN` token in their env variable and/or we've already hit our Github API rate limit.  

## How to test
Confirm CI builds successfully + the dev build spins up locally without any issues.
